metaflow/plugins/argo/test/test_single_run.py

```diff
@@ -0,0 +1,32 @@
+from metaflow.client.trigger_live_run import trigger_live_run  # trigger
+import time
+
+start_time = time.time()
+flow_name = "LiveRunSimpleFlow"
+# "LiveRunSimpleFlow", "LiveRunParamFlow", "LiveRunFailureFlow",
+# "LiveRunMissingFlow", "LiveRunFlowTriggeringFlow", None
+template_name = None
+# "liverunsimpleflow", "liverunparamflow", "liverunfailureflow",
+# "liverunmissingflow", "liverunflowtriggeringflow", None
```

Comment:
Please clean up commented out lines here before merging.

Reply:
Actually, should this file be removed entirely since you have all tests in `test_trigger_live_run.py`?

Reply:
Yes, I think removing is best. Will do.